sunbird_service_api_test/src/test/java/org/sunbird/integration/test/user/UserTest.java

```diff
@@ -329,6 +341,37 @@ public void validate(
             });
   }
 
+  @Test(dataProvider = "getUserByLoginIdFailure")
+  @CitrusParameters({"testName"})
+  @CitrusTest
+  public void testGetUserByLoginIdFailure(String testName) {
+    performPostTest(
+        testName,
+        TEMPLATE_DIR_GET_USER_BY_LOGIN_ID,
+        getLmsApiUriPath(GET_USER_BY_ID_SERVER_URI, GET_USER_BY_ID_LOCAL_URI),
+        REQUEST_JSON,
+        HttpStatus.BAD_REQUEST,
+        RESPONSE_JSON,
+        false,
+        MediaType.APPLICATION_JSON);
+  }
+
+  @Test(dependsOnMethods = {"testCreateUser"})
+  @CitrusTest
+  public void testGetUserByLoginIdSuccess() {
```

Comment:
This test need to be done after the dependent entities (e.g. user) is created using a common util.